go.work

```diff
@@ -4,9 +4,6 @@ toolchain go1.22.0
 
 use (
 	./aws-lambda-router
-	./composition-go
```

Comment:
I tend to keep them for better IDE integration. With the CI check we should be already safe.

Reply:
Alright, let's keep it simple for now. Just a check to avoid having unpredictable dependency updates.